### PR TITLE
Flesh out the irc channels

### PR DESCRIPTION
**Pull request by erickt**
This promotes #rust-beginners to a main channel, adds some general international channels (please help with transalations!), and sorts the topical channels.

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -74,37 +75,49 @@ Rust's developers coordinate in [#rust-internals][internals_irc]. It is for real
 - [#rust-lang][lang_irc] is the home of the [language team][language_team]
 - [#rust-community][community_irc] is the home of the [community team][community_team]
 
+### International channels
+
+- [#rust-br][ru_irc] é dedicado à programação de Ferrugem no Brasil
+- [#rust-de][ru_irc] ist für die allgemeine Diskussion über Rust auf Deutsch
```
Comment (skade) [inline community.md]: should be `[ru_de]` :)

File: community.md [hunk 2]
```diff
@@ -74,37 +75,49 @@ Rust's developers coordinate in [#rust-internals][internals_irc]. It is for real
...
+- [#rust-es][es_irc] es para una discusión general sobre Rust en alemán
+- [#rust-fr][fr_irc] est dédié à la programmation en Rouille et bon français
```
Comment (skade) [inline community.md]: Should `Rust` really be translated?

File: community.md [hunk 3]
```diff
@@ -74,37 +75,49 @@ Rust's developers coordinate in [#rust-internals][internals_irc]. It is for real
 - [#rust-lang][lang_irc] is the home of the [language team][language_team]
 - [#rust-community][community_irc] is the home of the [community team][community_team]
 
+### International channels
+
+- [#rust-br][ru_irc] é dedicado à programação de Ferrugem no Brasil
+- [#rust-de][ru_irc] ist für die allgemeine Diskussion über Rust auf Deutsch
+- [#rust-es][es_irc] es para una discusión general sobre Rust en alemán
```
Comment (skade) [inline community.md]: "alemán" is german in spanish and that's probably not what we want

File: community.md [hunk 4]
```diff
@@ -74,37 +75,49 @@ Rust's developers coordinate in [#rust-internals][internals_irc]. It is for real
 - [#rust-lang][lang_irc] is the home of the [language team][language_team]
 - [#rust-community][community_irc] is the home of the [community team][community_team]
 
+### International channels
+
+- [#rust-br][ru_irc] é dedicado à programação de Ferrugem no Brasil
```
Comment (skade) [inline community.md]: Wouldn't that be "bazilian portuguese"? I'll ask my brazilian colleague tomorrow.

File: community.md (hunk 3 above)
Comment (erickt) [inline community.md]: Oops, thanks!

File: community.md (hunk 4 above)
Comment (erickt) [inline community.md]: I'm not actually quite sure. The topic suggests it was started by the [São Paulo](http://www.meetup.com/Rust-Sao-Paulo-Meetup/) community. @bltavares, is this your channel?

**Comment by erickt** [comment]
Re-asking at the top-level since I just force pushed. @bltavares, is #rust-br your channel? If so, could you help tweak the description?

File: community.md [hunk 5]
```diff
@@ -74,37 +75,49 @@ Rust's developers coordinate in [#rust-internals][internals_irc]. It is for real
 - [#rust-lang][lang_irc] is the home of the [language team][language_team]
 - [#rust-community][community_irc] is the home of the [community team][community_team]
 
+### International channels
+
+- [#rust-br][br_irc] é dedicado à programação de Ferrugem no Brasil
+- [#rust-de][de_irc] ist für die allgemeine Diskussion über Rust auf Deutsch
+- [#rust-es][es_irc] es para una discusión general sobre Rust en español
+- [#rust-fr][fr_irc] est dédié à la programmation en Rust et français
+- [#rust-ru][ru_irc] для общей дискуссии о ржавчине на русском языке
```
Comment (petrochenkov) [inline community.md]: Minimal fix: [#rust-ru][ru_irc] для общих дискуссий о Rust на русском языке

File: community.md [hunk 6]
```diff
@@ -74,37 +75,49 @@ Rust's developers coordinate in [#rust-internals][internals_irc]. It is for real
 - [#rust-lang][lang_irc] is the home of the [language team][language_team]
 - [#rust-community][community_irc] is the home of the [community team][community_team]
 
+### International channels
+
+- [#rust-br][br_irc] é dedicado à programação de Ferrugem no Brasil
```
Comment (bltavares) [inline community.md]: `Ferrugem` means [rust](https://en.wikipedia.org/wiki/Rust) in portuguese. That was a good translation, but not the correct semantics here (:

I would translate to `é dedicado à discussão sobre Rust no Brasil`

**Comment by bltavares** [comment]
@erickt I've adjusted the description for #rust-br

File: community.md [hunk 7]
```diff
@@ -74,37 +75,49 @@ Rust's developers coordinate in [#rust-internals][internals_irc]. It is for real
...
+- [#rust-fr][fr_irc] est dédié à la programmation en Rust et français
+- [#rust-ru][ru_irc] для общей дискуссии о ржавчине на русском языке
+
 ### Topical channels
 
-- [#rust-beginners][beginners_irc] is a space for those new to Rust, lower traffic than #rust
 - [#cargo][cargo_irc] is for discussion of Cargo, Rust's package manager
-- [#servo][servo_irc] is for discussion of Servo, the browser engine written in Rust
-- [#rust-offtopic][offtopic_irc] is for general chit-chat amongst Rustaceans
+- [#rust-bots][bots_irc] notifcations about Rust from a selection of bots
```
Comment (skade) [inline community.md]: notifcations -> notifications

**Comment by steveklabnik** [comment]
I would like this to land before I re-do https://github.com/rust-lang/rust-www/pull/296 , so that they don't trample on each other.

**Comment by brson** [comment]
lgtm but needs tweaks

**Comment by erickt** [comment]
Thanks @skade, @bltavares and @petrochenkov! I updated the PR with your suggestions.

**Comment by bltavares** [comment]
:+1:

**Comment by skade** [comment]
:+1:

**Pull request merged**